src/client/app/inventory/item.popover.ts

```diff
@@ -56,7 +56,7 @@ export class InventoryItemPopover {
   }
 
   salvage() {
-    this.socketService.emit(ServerEventName.ItemSalvage, { itemId: this.item.id });
+    this.socketService.emit(ServerEventName.ItemSalvage, { itemId: this.item.id, ItemSlot: null });
```

Comment:
this should probably pass in `itemSlot`, not `ItemSlot`

Reply:
Indeed, a small typo on my part. It's fixed now.